src/node.rs

```diff
@@ -244,6 +244,18 @@ impl<S: StorageExt> SawtoothRaftNode<S> {
 
         let is_leader = self.raw_node.raft.state == raft::StateRole::Leader;
         if is_leader {
+            //check if we stepped up as leader from follower state
+            if self.follower_state.is_some() {
```

Comment:
This can be rewritten more simply: `if let Some(FollowerState::Committing(ref block_id)) = self.follower_state { ... }`

Reply:
Updated the commit